File: onboard/onboarding.py
import os
from acontext import AcontextClient

client = AcontextClient(
    api_key=os.getenv("ACONTEXT_API_KEY"),
)

print(client.ping())

session = client.sessions.create()
client.sessions.store_message(
    session_id=session.id,
    blob={
        "role": "assistant",
        "content": """Here is my plan:
1. Use Next.js for the frontend
2. Use Supabase for the database
3. deploy to Cloudflare Pages
""",
    },
)
client.sessions.store_message(
    session_id=session.id,
    blob={
        "role": "user",
        "content": "Confirm, go ahead. Use tailwind for frontend styling.",
    },
)


messages = client.sessions.get_messages(session_id=session.id)
print(messages.items)

File: onboard/onboarding.ts

import { AcontextClient } from "@acontext/acontext";

const client = new AcontextClient({
    apiKey: process.env.ACONTEXT_API_KEY,
});

async function main() {
    console.log(await client.ping());

    const session = await client.sessions.create();
    await client.sessions.storeMessage(session.id, {
        role: "assistant",
        content: `Here is my plan:
1. Use Next.js for the frontend
2. Use Supabase for the database
3. deploy to Cloudflare Pages
`,
    });
    await client.sessions.storeMessage(session.id, {
        role: "user",
        content: "Confirm, go ahead. Use tailwind for frontend styling.",
    });

    const messages = await client.sessions.getMessages(session.id);
    console.log(messages.items);
}

main();

